### PR TITLE
update Readme

### PR DESCRIPTION
**Pull request by jay-saldanha**
sync with the gdoc final edits done on Apr8

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/1158)

### CODE REVIEW

File: components/gcp/dataproc/create_cluster/README.md [hunk 1]
```diff
@@ -1,89 +1,94 @@
 
-# Creating a Cluster with Cloud Dataproc
-A Kubeflow Pipeline component to create a cluster in Cloud Dataproc service.
+# Name
+Data processing by creating a cluster in Cloud Dataproc
```
Comment (animeshsingh) [inline components/gcp/dataproc/create_cluster/README.md]: Should be something like - 'Create a Cloud Dataproc cluster for data processing' ?

File: components/gcp/dataproc/create_cluster/README.md [hunk 2]
```diff
@@ -97,38 +102,29 @@ def dataproc_create_cluster_pipeline(
...
-        name=name, 
-        name_prefix=name_prefix, 
-        initialization_actions=initialization_actions, 
-        config_bucket=config_bucket, 
-        image_version=image_version, 
-        cluster=cluster, 
-        wait_interval=wait_interval).apply(gcp.use_gcp_secret('user-gcp-sa'))
+    dataproc_create_cluster_op(project_id, region, name, name_prefix, job_name_prefix, initialization_actions, 
+       config_bucket, image_version, cluster, wait_interval).apply(gcp.use_gcp_secret('user-gcp-sa'))
 ```
```
Comment (animeshsingh) [inline components/gcp/dataproc/create_cluster/README.md]: General convention in other files it to split the variables on their own individual lines

File: components/gcp/dataproc/create_cluster/README.md [hunk 3]
```diff
@@ -142,3 +138,13 @@ experiment = client.create_experiment(EXPERIMENT_NAME)
 run_name = pipeline_func.__name__ + ' run'
 run_result = client.run_pipeline(experiment.id, run_name, pipeline_filename, arguments)
 ```
+
+## References
+*   [Kubernetes Engine for Kubeflow](https://www.kubeflow.org/docs/started/getting-started-gke/#gcp-service-accounts)
```
Comment (animeshsingh) [inline components/gcp/dataproc/create_cluster/README.md]: Not sure if the text and hyperlink co-relate?

File: components/gcp/dataproc/create_cluster/README.md (hunk 1 above)
Comment (jay-saldanha) [reply components/gcp/dataproc/create_cluster/README.md]: the title is according to the Cloud docs style guidelines.

File: components/gcp/dataproc/create_cluster/README.md (hunk 3 above)
Comment (jay-saldanha) [reply components/gcp/dataproc/create_cluster/README.md]: yes, we can remove this hyperlink, doesn't add any value

File: components/gcp/dataproc/create_cluster/README.md (hunk 2 above)
Comment (jay-saldanha) [reply components/gcp/dataproc/create_cluster/README.md]: I leave it up to @hongye-sun to decide, I don't have any preference on this.

**Comment by jay-saldanha** [comment]
Thanks for the review @animeshsingh

**Comment by jay-saldanha** [comment]
yes please go ahead


*Jayana Saldanha | *Technical Writer (Akorbi) | Cloud AI Platform Docs

**Comment by vicaire** [comment]
Closing as instructed by @jay-saldanha

**Comment by vicaire** [comment]
Closing as instructed.

**Pull request closed without merging**